### PR TITLE
Add commonbaseline mathod

### PR DESCRIPTION
**Pull request by zouyuxin**

### CODE REVIEW

File: R/data2cov.R [hunk 1]
```diff
@@ -12,8 +12,14 @@ cov_pca = function(data,npc,subset = NULL){
   assert_that(npc<=n_conditions(data))
   if(is.null(subset)){subset = 1:n_effects(data)}
   res.svd = svd(data$Bhat[subset,],nv=npc,nu=npc)
-  if (!all(data$Shat == 1))
-    message("svd currently performed on Bhat; maybe should be Bhat/Shat?")
+  if (class(data) == 'mash'){
+    if (!all(data$Shat == 1))
+      message("svd currently performed on Bhat; maybe should be Bhat/Shat?")
+  } else{
```
Comment (stephens999) [inline R/data2cov.R]: can you make an explicit check for mash.contrast here? (then have an else that stops if class(data) is neither mash nor mash.contrast

Comment (stephens999) [reply R/data2cov.R]: and do the same wherever you have this kind of check?

Comment (zouyuxin) [reply R/data2cov.R]: Ok, I will add it.

Comment (zouyuxin) [reply R/data2cov.R]: I added the check for mash.contrast, and stop if the data is neither mash nor mash.contrast.

**Pull request closed without merging**